ui/app/AppLayouts/Profile/views/EnsListView.qml

```diff
@@ -230,6 +230,7 @@ Item {
                 amISender: false
                 sender.displayName: root.ensUsernamesStore.preferredUsername
                 sender.profileImage.assetSettings.isImage: true
+                sender.profileImage.assetSettings.color: Utils.colorForPubkey(root.ensUsernamesStore.pubkey)
```

Comment:
Isn't this our own pubkey? I think you can get that color in the global user store thingy.
NVM, I checked and it's not there, but we can easily add it in `user_profle.nim`. Up to you.

Reply:
If we consider adding it there, then it makes sense to find all other places where we could use it.
I don't think it makes sense to implement here or any time soon. We have much more other stuff to do 🙂